src/flypad-backend/src/Aircraft/AircraftProcedures.h

```diff
@@ -1,13 +1,16 @@
 #pragma once
 
-#include <map>
-#include <string>
 #include <vector>
-
-using namespace std;
+#include <algorithm>
+#ifdef DEBUG
+#include <iostream>
+#endif
+#include <array>
+#include <span>
+#include <numeric>
 
 struct ProcedureStep {
-  std::string description;
+  const char* description;
```

Comment:
If you are trying to make things more cpp-like I do not understand this change - I'd prefer to keep the std::string
That counts for all your changes from std:string to char*

Reply:
This way you can make every `std::array` `constexpr`. AFAIK `std::string` is `constexpr` in C++20, but at the time of development, I was still using C++17.